packages/kbn-config-schema/src/types/nullable_type.test.ts

```diff
@@ -24,16 +24,36 @@ test('returns value if specified', () => {
   expect(type.validate('test')).toEqual('test');
 });
 
-test('returns null if null', () => {
+test('returns null if null for string', () => {
   const type = schema.nullable(schema.string());
   expect(type.validate(null)).toEqual(null);
 });
 
-test('returns null if undefined', () => {
+test('returns null if null for number', () => {
+  const type = schema.nullable(schema.number());
+  expect(type.validate(null)).toEqual(null);
+});
+
+test('returns null if null for boolean', () => {
```

Comment:
could you add a couple of tests validating it returns value for other types different from null?
```js
const type = schema.nullable(schema.boolean());
expect(type.validate(true)).toBe(true);
```

Reply:
added some additional tests - validating positively for more basic types, and negatively for a property in an object type